### ISSUE TITLE
Report not viewable in Firefox 4

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#113]
```
Since upgrading to Firefox 4, reports have the same problems they've had in 
Chrome/Safari.
```

Original issue reported on code.google.com by `sarma...@gmail.com` on 24 Mar 2011 at 3:52

Attachments:
- [Skipfish - scan results browser - Mozilla Firefox_001.png](https://storage.googleapis.com/google-code-attachments/skipfish/issue-113/comment-0/Skipfish - scan results browser - Mozilla Firefox_001.png)

**Comment by GoogleCodeExporter** [comment]
```
This will be fixed in the next version, cheers!
```

Original comment by `niels.he...@gmail.com` on 25 Feb 2012 at 8:42
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Reports open fine in Firefox 10; time to upgrade :)
```

Original comment by `hlubo...@gmail.com` on 3 Mar 2012 at 3:57

**Comment by GoogleCodeExporter** [comment]
```
We aim to fix this in the next release as part of a general change to improve 
offline reading (as well in chrome)
```

Original comment by `niels.he...@gmail.com` on 5 Apr 2012 at 2:46

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.06b. Thanks for the report!
```

Original comment by `niels.he...@gmail.com` on 13 May 2012 at 10:43
- Changed state: **Fixed**
